spec/src/main/asciidoc/core/scopescontexts.asciidoc

```diff
@@ -228,6 +228,34 @@ From time to time, the container must obtain an _active context object_ for a ce
 
 If there is exactly one active instance of `Context` associated with the scope type, we say that the scope is _active_.
 
+==== Activating Built In Contexts
```

Comment:
I just want to check with everyone.  Is this in the right place? Is it enough detail?